### DISCUSSION TITLE
BUG? - No invite in latest version =(

### DISCUSSION THREAD

**Discussion by Dracool2006** [m4dEngi/RemotePlayWhatever#21] [General]
There is no invite to Remote Play in new version. 

Remote Play Steam application starts correctly, friend added correctly, but there he can't connect to game cuz he didn't see connect button in steam chat overlay

**Comment by pitmetak** [comment]
Same here, except I don't seem to get an option to appear to even invite my friends to the game.

**Comment by m4dEngi** [comment]
You need to be on stable client branch and signed in to friends network with in-game status for RPW to work.
